src/components/Select/AutosuggestMulti/SuggestionTag/SuggestionTag.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { MdClose } from 'react-icons/md';
+import { bem } from '../../../../utils';
+import styles from './SuggestionTag.scss';
+
+const { block, elem } = bem('SuggestionTag', styles);
+
+type Props = {
+    /** Tag Stretches on full width */
+    isStretched?: boolean;
+    /** Tag has restricted width */
+    isBounded?: boolean;
```

Comment:
```suggestion
    isStretched?: boolean;
    isBounded?: boolean;
```

these should have a default value of `false` as usual